README.md

```diff
@@ -123,11 +124,7 @@ questions, from making better predictions of species distribution
 [@Cazelles2016Theorya] to forecasting the impact of climate change on ecological
 webs [@Gilman2010FraCom].
 
-The lack of clear guidelines on the use of probabilistic interaction data is
-worrisome, both for data producers and re-users who generate and manipulate
-these numbers. This is concerning because sampling strategies and decisions
-regarding network construction can affect our understanding of network
-properties [@Brimacombe2023ShoReu]. Besides methodological difficulties that may
+Beyond methodological difficulties that may 
```

Comment:
Not sure about this (I moved the two first sentences later). To me, the topic sentence and main/first idea of the paragraph should be the lack of definition of probabilistic interactions. It's the most important element to introduce. The lack of guidelines should then come in later.